downloaders/apkmirror-downloader/src/main/kotlin/app/revanced/manager/plugin/downloader/apkmirror/APKMirrorDownloader.kt

```diff
@@ -0,0 +1,106 @@
+@file:Suppress("Unused")
+
+package app.revanced.manager.plugin.downloader.apkmirror
+
+import android.net.Uri
+import android.os.Parcelable
+import android.util.Log
+import app.revanced.manager.plugin.downloader.DownloadUrl
+import app.revanced.manager.plugin.downloader.Downloader
+import app.revanced.manager.plugin.downloader.download
+import app.revanced.manager.plugin.downloader.webview.runWebView
+import app.revanced.manager.plugin.utils.Merger
+import com.reandroid.apk.APKLogger
+import kotlinx.parcelize.Parcelize
+import java.io.File
+import java.net.URI
+import java.nio.file.Files
+import java.util.UUID
+import java.util.zip.ZipFile
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.deleteRecursively
+import kotlin.io.path.outputStream
+
+@Parcelize
+class ApkMirrorApp(
+    val downloadUrl: DownloadUrl
+) : Parcelable
```

Comment:
This type is unnecessary because `DownloadUrl` implements `Parcelable` and can be returned directly.